client/state/selectors/get-post-revision-changes.js

```diff
@@ -23,14 +23,21 @@ const getPostRevisionChanges = createSelector(
 		const orderedRevisions = getPostRevisions( state, siteId, postId, 'display' );
 		const revisionIndex = findIndex( orderedRevisions, { id: revisionId } );
 		if ( revisionIndex === -1 ) {
+			console.log( 'empty diff' );
 			return { content: [], title: [] };
 		}
 		const previousRevision = orderedRevisions[ revisionIndex + 1 ];
 		const currentRevision = orderedRevisions[ revisionIndex ];
-		return {
+		const diffStartMoment = Date.now();
+		const changes = {
 			content: diffKey( 'content', previousRevision, currentRevision ),
 			title: diffKey( 'title', previousRevision, currentRevision ),
 		};
+		const diffEndMoment = Date.now();
+
+		console.log( { diffDuration: diffEndMoment - diffStartMoment } );
```

Comment:
I'm thinking about tracking the diff time here - it seems to give good results but it feels like a naive approach.